Dockerfile

```diff
@@ -5,21 +5,3 @@ LABEL maintainer="Shaun Jackman <sjackman@gmail.com>" \
       org.label-schema.url="http://www.bcgsc.ca/services/orca" \
       org.label-schema.vcs-url="https://github.com/bcgsc/orca" \
       org.label-schema.vendor="BC Cancer Genome Sciences Centre"
-
-RUN sudo apt-get update \
-    && sudo apt-get install -y --no-install-recommends man-db \
-    && sudo rm -rf /var/lib/apt/lists/*
```

Comment:
Can you please add `man-db` back to `orca-0/Dockerfile`? I forget why we removed it though, and the PR doesn't say. https://github.com/bcgsc/orca/pull/82

Reply:
other than it failed to install. I think it conflicted with another package.

Reply:
I did go back and add `man-db` to `orca-0/Dockerfile` in a previous PR - it should be there

Reply:
It doesn't look like `man-db` is there, but `vim` is there.

Reply:
Here, right? ```# fonts-dejavu-core is required by graphviz.
RUN sudo apt-get update \
        && sudo apt-get install -y --no-install-recommends \
                fonts-dejavu-core man-db \
        && sudo rm -rf /var/lib/apt/lists/*
```

Reply:
I believe the `brew install man-db` wasn't working, so we added it in this way

Reply:
Ah, I see. You can remove it from `apt-get` and add it to `brew install`.

Reply:
Sounds good -- done!

Reply:
Thanks!